data/language/dutch.txt

```diff
@@ -3368,9 +3368,9 @@ STR_3366    :{BLACK}= Attractie
 STR_3367    :{BLACK}= Eetkraampje
 STR_3368    :{BLACK}= Drankkraampje
 STR_3369    :{BLACK}= Souvenirkraampje
-STR_3370    :{BLACK}= Infokiosk
+STR_3370    :{BLACK}= Informatiekiosk
 STR_3371    :{BLACK}= Eerste hulp
-STR_3372    :{BLACK}= PIN
+STR_3372    :{BLACK}= Geldautomaat
```

Comment:
PIN staat voor pincode, waar ATM staat voor Automated Teller Machine. Geldautomaat is misschien niet ideaal, maar ik kan echt niets beters bedenken, bovendien blijft het korter dan eerder genoemden en zou het geen problemen op moeten leveren.